.changeset/quiet-pears-float.md

```diff
@@ -2,4 +2,7 @@
 '@apollo/server': patch
 ---
 
-Update schemaIsSubgraph to also support non nullable \_Service.sdl
+The subgraph spec has evolved in Federation v2 such that the type of
+`_Service.sdl` (formerly nullable) is now non-nullable. Apollo Server now
+detects both cases correctly in order to determine whether to install / enable
```

Comment:
It does also affect some warning/error check stuff about usage and schema reporting. Is that worth mentioning?

Reply:
8f4b920e3